lib/rucio/core/transfer.py

```diff
@@ -45,9 +46,10 @@
 REGION_SHORT = make_region().configure('dogpile.cache.memcached',
                                        expiration_time=600,
                                        arguments={'url': "127.0.0.1:11211", 'distributed_lock': True})
+USER_TRANSFERS = config_get('conveyor', 'user_transfers', False, None)
 
 
-def submit_bulk_transfers(external_host, files, transfertool='fts3', job_params={}, timeout=None):
+def submit_bulk_transfers(external_host, files, transfertool='fts3', job_params={}, timeout=None, user_transfer_job=False):
```

Comment:
I would keep this change on the same line of activity mapping comment. So, putting it on a dedicated ticket for the second iteration to avoid forgetting it, while not pushing for the inclusion on this PR.